FlashCardApp/UI/ProgramController.cs

```diff
@@ -0,0 +1,535 @@
+using FlashCardApp.Data;
+using FlashCardApp.DTO;
+using FlashCardApp.Input;
+using FlashCardApp.Models;
+
+namespace FlashCardApp.UI;
+
+public static class ProgramController
+{
+    private static readonly IDatabaseManager DbManager = new SqlServerDatabaseManager();
+    private static readonly UserInput Input = new();
+    private static readonly TableVisualizationEngine DisplayTable = new();
+
+    private static void ViewMainMenu()
+    {
+        Console.WriteLine("MAIN MENU");
+        Console.WriteLine("-------------------------------------");
+        Console.WriteLine("What would you like to do?\n");
+        Console.WriteLine("study to go to Study Area");
+        Console.WriteLine("settings to go to Settings");
+        Console.WriteLine("exit to End Program");
+        Console.WriteLine("\nType your choice and hit Enter");
+        Console.Write("Your choice? ");
+    }
+
+    private static void CreateTables()
+    {
+        DbManager.CreateStackTable();
+        DbManager.CreateFlashCardTable();
+        DbManager.CreateStudyAreaTable();
+    }
+
+    public static void StartProgram()
+    {
+        CreateTables();
+
+        ViewMainMenu();
+        var choice = Input.GetChoice();
+
+        while (choice != "exit")
+        {
+            switch (choice)
+            {
+                case "study":
+                    ManageStudyArea();
+                    break;
+                case "settings":
+                    ManageStacksSettings();
+                    break;
+                default:
+                    Console.Clear();
+                    Console.WriteLine("Wrong input!");
+                    break;
+            }
+
+            ViewMainMenu();
+            choice = Input.GetChoice();
+        }
+    }
+
+    // Study Area Operations
+    private static void ViewStudyAreaMenu()
+    {
+        Console.WriteLine("STUDY AREA\n");
```

Comment:
I would personally would ask the user to type one character just to save some key presses, but again it's my personal preference.